### DISCUSSION TITLE
关于异地内网网地址的代理

### DISCUSSION THREAD

**Discussion by vincenthsing** [nikkinikki-org/OpenWrt-nikki#343] [Q&A] [answered]
在云服务器部署了v2ray，内网网段为172.12.66.x。
配置文件中添加 172.12.66.0/24 到云服务器v2ray节点的规则，但发现无法正常通过内网地址访问到云服务器。相同配置在openclash下删除内网网段绕过后可以正常使用。

在这个插件没看到有绕过的配置，是不是默认绕过了所有内网网段呢？谢谢大佬们

**Comment by morytyann** [comment, marked as answer]
在编辑器里

**Reply by vincenthsing** [reply]
感谢大佬及时的回答，试了非常好用，比起oc那些对我没用的配置简直太清爽了，感谢感谢！！
